### PR TITLE
Ap 1598 offline accounts non-passported

### PR DESCRIPTION
**Pull request by willc-work**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1598)

Added 2 new pages to the passported journey and updated the flow after vehicles pages to cover both passported and non-passported journeys.

Added tests for the new pages and updated existing tests to cover the 2 new flows.

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/requests/providers/client_bank_accounts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  let(:applicant) { create :applicant }
+  let!(:bank_provider) { create :bank_provider, applicant: applicant }
+  let!(:bank_account) { create :bank_account, bank_provider: bank_provider }
+  let(:legal_aid_application) { create :legal_aid_application, :with_non_passported_state_machine, :with_savings_amount, applicant: applicant }
+  let!(:savings_amount) { create :savings_amount }
+  let(:application_id) { legal_aid_application.id }
+  let!(:provider) { legal_aid_application.provider }
+
+  describe 'GET providers/client_bank_account' do
+    # subject { get providers_legal_aid_application_client_bank_account_path(legal_aid_application.id) }
```
Comment (mpw5) [inline spec/requests/providers/client_bank_accounts_spec.rb:13]: Remove this comment? Or remove the one below and enable this one?

**Review by mpw5 — APPROVED** [review]
Couple of comments, but not blockers. Excellent work.

Just checking, as a result of this will all saving/current accounts be sent to CFE for the means calculation? There are no further changes to Apply or CFE to get that to work?

File: app/controllers/providers/client_bank_accounts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+module Providers
+  class ClientBankAccountsController < ProviderBaseController
```
[the remark this answers is not in the document]
Comment (willc-work) [reply app/controllers/providers/client_bank_accounts_controller.rb:2]: amended

**Review by mpw5 — APPROVED** [review]
Re-approving after changes 🎉

**Pull request merged**